src/client/opamCommands.ml

```diff
@@ -851,14 +851,16 @@ let config ?(setopt=false) () =
     "Without argument, prints a documented list of all available variables. \
      With $(i,PACKAGE), lists all the variables available for these packages. \
      Use $(i,-) to include global configuration variables for this switch.";
-    "set-var", `set_var, ["[switch|global]"; "NAME"; "[VALUE]"],
+    "set-var", `set_var, ["<switch|global>"; "NAME"; "[VALUE]"],
     "Set the given variable globally or in the current switch. Warning: \
      changing a configured path will not move any files! This command does not \
      perform any variable expansion.";
-    "set-opt", `set_opt, ["[switch|global]"; "FIELD[(+=|-=|=)VALUE]"],
-    "Set the given opam configuration field in the global/switch configuration \
-     file.  If $(b,op VALUE) is omitted, $(b,FIELD) is reset to its default \
-     initial configuration, as after a fresh init (use `opam init \
+    "option", `option, ["<switch|global>"; "[FIELD[(=|+=|-=)VALUE]"],
+    "Global and switch configuration options settings. If no field is given, \
+     list all configurable fields. If a only a field name is given, display \
+     its content. Otherwise, set the given field in the global/switch \
+     configuration file. If $(b,VALUE) is ommited, $(b,FIELD) is reset to its \
```

Comment:
`set or update`
`omitted`
But I would say `empty` instead, since the docstring doesn't show it as optional